packages/fxa-settings/src/components/App/en-US.ftl

```diff
@@ -20,6 +20,7 @@
 # “Account” can be localized, “Firefox” must be treated as a brand.
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
+-product-firefox-sync = Firefox Sync
```

Comment:
These are called "terms": https://projectfluent.org/fluent/guide/references.html

Reply:
@LZoog 

Sync is not a feature name, it's been deprecated since FIrefox 89. We shouldn't be exposing a term for it.
https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Word+List#WordList-sync